src/main/java/entities/InvitationFactory.java

```diff
@@ -0,0 +1,7 @@
+package entities;
+
+public class InvitationFactory implements InvitationFactoryInterface {
+    @Override
+    public Invitation create(User toUser, User fromUser, Task task){return new Invitation(toUser, fromUser, task);}
```

Comment:
Is there going to be multiple invitations that use the same parameters, and is that why you are using a factory design pattern? Right now the factory seems kind of unnecessary.